napari_organoid_counter/_widget.py

```diff
@@ -112,9 +120,84 @@ def __init__(self,
+                        # Update to the new color
+                        current_edge_colors[idx] = settings.COLOR_CLASS_1
+                        changed_indices.append(idx)  # Add the index to the list
+                    self.cur_shapes_layer.edge_color = current_edge_colors  # Apply the changes
+                    show_info(f"Changed edge color of shapes {changed_indices} to green.")
+                else:
+                    show_warning("No shapes selected to change edge color.")
+
+        # Key binding to change the edge_color of the bounding boxes to blue
+        @self.viewer.bind_key('h')
+        def change_edge_color_to_blue(viewer: napari.Viewer):
+            if not self.multi_annotation_mode:  # Check if single-annotation mode is active
+                show_error("Cannot change edge color. Change to multi-annotation mode to enable this feature.")
+                return         
+            if self.cur_shapes_layer is not None:  # Ensure shapes layer exists
+                selected_shapes = self.cur_shapes_layer.selected_data
+                if len(selected_shapes) > 0:
+                    # Modify the edge color only for the selected shapes
+                    current_edge_colors = self.cur_shapes_layer.edge_color
+                    changed_indices = [] # To collect all changed indices
+                    for idx in selected_shapes:
+                        # Save original color
+                        # if idx not in self.original_colors: 
+                            # self.original_colors[idx] = current_edge_colors[idx].copy()
+                        # Update to the new color
+                        current_edge_colors[idx] = settings.COLOR_CLASS_2
+                        changed_indices.append(idx)  # Add the index to the list
+                    self.cur_shapes_layer.edge_color = current_edge_colors  # Apply the changes
+                    show_info(f"Changed edge color of {changed_indices} to blue.")
+                else:
+                    show_warning("No shapes selected to change edge color.")
+
+        # Key binding to reset the edge_color of selected bounding boxes to the original magenta color
+        @self.viewer.bind_key('m')
+        def change_to_original_color(viewer: napari.Viewer):
+            if not self.multi_annotation_mode:  # Check if single-annotation mode is active
+                show_info("Cannot change edge color. Change to multi-annotation mode to enable this feature.")
+                return
+            if self.cur_shapes_layer is not None:  # Ensure shapes layer exists
+                selected_shapes = self.cur_shapes_layer.selected_data
+                if len(selected_shapes) > 0:
+                    current_edge_colors = self.cur_shapes_layer.edge_color
+                    # Set the edge color back to magenta 
+                    new_edge_color = [1., 0, 1., 1.]  # RGBA for magenta
```

Comment:
We can define also magenta in the settings.py file. Then I would also change the boxes definition in line 298, and use the color definition there as well. You can call the variable  COLOR_DEFAULT.